### ISSUE TITLE
Investigate / solve errant reassignment of switch == Constants.round_values[0] to '0' in pages.py or Game 3 and Game 4

### ISSUE THREAD

**Issue by ianchadd** [ianchadd/rpi_3208#279]
**Problem**
Lines 29-30 of Game 4/pages.py and lines 30-31 of Game 3/pages.py take the switch and reassign it to 0 if it is equal to the lowest possible tournament prize (e.g. 0.25). Is this necessary? We actually **do** want to know whether the subject always chose the tournament (which is true here), but it is technically incorrect to say that the switch is equal to 0 (as 0 is not an available tournament prize).

**Notes**
Jack mentioned that this might be because of the python logic that checks the switch point against the randomly chosen tournament value for games 3 and 4, which appears to be using the strict inequality at present, but we actually do want the weak inequality. For example, if the subject chooses tournament at a price of $1.00, but not at 0.75, the switch should be equal to 1.00 **and** if the tournament prize is randomly chosen to be 1.00, they should enter the tournament.

**Comment by mcdaij** [comment]
I've scoured the code for any references to this variable (or any of the many duplicates made of this variable) to find any possible reason for it to need to be set to '0'. I have not been able to find any code that depends on the value being lower than the actually selected switch value, or in fact, on it being '0'. I've removed the line, and the only test case that actually worried me (where it randomly selects the minimum tournament payout) appears to work. (I've also tested it with other cases and those appear to work as well). 
![image](https://user-images.githubusercontent.com/68068255/94373146-72da3500-00d1-11eb-8689-bfdb6fd9a10f.png)
That said, in the code there's a lot of redundancy, a lot of code that doesn't appear to serve any practical purpose, and a lot of code that doesn't appear to ever be triggered, so it's difficult to actually figure out what is used when and where, and I can't be certain.
